### ISSUE TITLE
i3 replaces my wallpaper with a screenshot

### ISSUE THREAD

**Issue by psychon** [i3/i3#4371]
## I'm submitting a…

<pre>
[x] Bug
[ ] Feature Request
[ ] Documentation Request
[ ] Other (Please describe in detail)
</pre>

[I removed most of the rest of the template since it makes more sense to start with the reproduction instructions. I hope that's okay.]

## What I did

I read some source code and wondered "wait, what?!". Then I did the following just so to trigger this issue.
```sh
Xephyr -listen tcp :1  & DISPLAY=:1 sh -c '(sleep 1 ; xterm & sleep 1 ; (sleep 1 ; pkill xterm) & i3 -c /dev/null)'
```
Result:
![foo](https://user-images.githubusercontent.com/89482/110627135-e19fdc00-81a1-11eb-9f43-51dfeb3e1bed.png)

## What is wrong

That's not an xterm. That's a screenshot of an xterm. As my wallpaper.

## How to make this into a slightly less weird issue

Imagine I set my wallpaper not from the i3 config, but before starting i3. Together with some transparency, I bet I could turn this into an actual "bad rendering" problem...

## Why this happens

Edit: This goes back to 4f5e0e794 from 2013.

https://github.com/i3/i3/blob/af2171d01d40613ec8e6eddcc148722b0ec248f9/src/main.c#L979-L998

(The above is btw not the best way to set a wallpaper since one cannot query the background of a window. Hence, wallpaper setters are expected to set the `ESETROOT_PMAP_ID` and/or `XROOTPMAP_ID` properties of the root window. But doing this properly requires a separate X11 connection since you have to actually leak the pixmap so that others can still use it. Which means you need to `SetCloseDownMode` to prevent the X11 server from destroying the pixmap when you disconnect. It is complicated.)

## What I would do about this

Just remove that piece of code and tell people to properly set a wallpaper.

## Environment

Output of `i3 --moreversion 2>&-`:
<pre>
i3 version: 
Binary i3 version:  4.19.1-non-git © 2009 Michael Stapelberg and contributors
Running i3 version: 4.19.1-non-git (pid 709)
Loaded i3 config: /dev/null (Last modified: Sa 06 Mär 2021 08:41:19 CET, 361408 seconds ago)

The i3 binary you just called: /usr/bin/i3
The i3 binary you are running: i3
</pre>
Why the heck does it claim that my config is `/dev/null`? That's the last config that I loaded, but not in this X11 server.... ARGH, it broke my IPC socket again. Unrelated to this issue, but: How can I start more than one i3 process without having my IPC socket break all the time? :-(

<details><summary>Config file</summary><pre>
I explicitly used /dev/null, just for this template.
</pre>
</details>

<pre>
Logfile URL:
</pre>

<pre>
- Linux Distribution & Version: Debian testing
- Are you using a compositor (e.g., xcompmgr or compton): no
</pre>

**Comment by i3bot** [comment]
I don’t see a link to logs.i3wm.org. Did you follow https://i3wm.org/docs/debugging.html? (In case you actually provided a link to a logfile, please ignore me.)

**Comment by orestisfl** [comment]
Also see https://github.com/i3/i3/issues/2869 (and I swear we had yet another issue around this code path but I can't find it)

**Comment by psychon** [comment]
Ah, thanks. And the newest comment in there is even from me... :-)

I'll send a PR that removes that piece of code and will see what happens.
